search.html

```diff
@@ -286,6 +287,38 @@ <h2><a class="anchor" id="keyvalue"></a>Key-Value Pair Annotation Search</h2>
 
 <p>will return objects with a Key-Value pair annotation with the specific key: <span class="code-example">Temperature</span>.</p>
 
+<h2><a class="anchor" id="advanced"></a>Advanced Search</h2>
+
+<p>Advanced search is available in OMERO.web only.</p>
+
+<p>Inside the search page, click on the <span class="bold">Advanced</span> tab in the left-hand pane.</p>
+
+<p>OMERO.server uses <a href="https://lucene.apache.org/">Lucene</a> to perform the search commands. The <span class="bold">Advanced</span> tab accepts directly the Lucene queries, with no parsing. Follow the steps below to get started with Lucene queries.
+
+<p>Construct searches for a string in any <span class="bold">Field</span> as listed in <a href="https://docs.openmicroscopy.org/latest/omero/developers/Modules/Search.html#field-names"
```

Comment:
I just noticed that our docs say you can also search for text without specifying a field, so that it covers all fields. Do we want to include that or is that less useful?
I wonder if users (even Advanced users) would be familiar with the meaning of "string"? I forget what terms I knew before being a developer.
Would it be nicer to provide a more user-friendly version of the Fields table from our docs, listing all the fields where we can think of use-cases, and omitting things like ```file.sha1``` and ```details.updateEvent.id```. This would save users from having to go elsewhere and try to find the useful fields amongst the list, although we could still include the link "to the full list for reference". The only downside is that we have to maintain 2 lists, but the Lucene indexing doesn't change very often.